### DISCUSSION TITLE
SoftBody3D doesn't work with Godot 4.2.1 and Jolt 0.12.0

### DISCUSSION THREAD

**Discussion by JohnnyRouddro** [godot-jolt/godot-jolt#797] [Questions] [answered]
When using Jolt, all the variables become 0 and unmodifiable. Works fine in Godot Default Physics.

https://github.com/godot-jolt/godot-jolt/assets/30183816/6a838f20-b0e8-4824-989e-aba9b4c0e565

**Comment by mihe** [comment, marked as answer]
No release has been published with the implementation for soft bodies yet. You'll need to [build](https://github.com/godot-jolt/godot-jolt/blob/master/docs/building.md) latest `master` from source in order to try that out. Note that they're still somewhat untested though.

I had hoped to push a new release by now, but there's other stuff in `master` that needs more testing/fixing before that can happen, and unfortunately I'm pretty swamped as it is.

**Reply by JohnnyRouddro** [reply]
Oops, my bad then. Thanks for the quick reply. Will be waiting for the next release. Thanks for all the hard work! Godspeed 🖖
